demo/flutter/lib/main.dart

```diff
@@ -48,7 +44,9 @@ class _MyAppState extends State<MyApp> {
   }
 
   Future<void> initRhino() async {
-    String contextPath = await _extractDemoContext();
+    String platform = Platform.isAndroid ? "android" : "ios";
```

Comment:
This is pedantic, but explicitly check for "isIos"

I've seen this type of logic blow up in weird ways before when something that was originally 2 choices increases to 3, or some other underlying assumption changes. We can simply throw if it is a thirdt ype for now with and then output the $platform in the error string.